src/Designer/frontend/packages/shared/src/types/Policy.ts

```diff
@@ -1,19 +1,9 @@
-export type Action =
-  | 'read'
-  | 'write'
-  | 'delete'
-  | 'confirm'
-  | 'sign'
-  | 'instantiate'
-  | 'reject'
-  | 'pay'
-  | 'complete';
-
 export type PolicyRule = {
   ruleId: string;
   description: string;
   subject: Array<string>;
-  actions: Action[];
+  actions: string[];
```

Comment:
I had to change this to `string` since that's the type of the corresponding property in the deleted `PropertyRule` type. I tried keeping `Action` and changing other types to align with it, but discovered that there was test data with strings that did not match any of the `Action` types, like `'consent'` and `'requestconsent'`.